scheduler/src/cook/mesos.clj

```diff
@@ -142,8 +142,10 @@
                 rebalancer-trigger-chan straggler-trigger-chan]} trigger-chans
         {:keys [hostname server-port server-https-port]} server-config
         datomic-report-chan (async/chan (async/sliding-buffer 4096))
-
+        ; TODO: Tech-debt. Should remove compute-cluster when we rethink initialization and leadership.
+        ; TODO: See also cluster-leadership-chan below.
         compute-cluster (cc/get-default-cluster-for-legacy)
```

Comment:
(Documenting a discussion) I proposed holding off on this refactor/cleanup; we need to rethink leader election: in the new multiple compute cluster world, we might not want cook to go down if any single compute cluster goes bad. We need to decide on what the new policy should be, then implement it. Thus, we would be visiting this code in about a week or so. 

The resolution of the discussion was to do this cleanup/refactor as part of this change, but implement it so that we have no functionality changes. Then implement the new policy in a followup change.